lms/djangoapps/certificates/api.py

```diff
@@ -172,13 +173,22 @@ def generate_user_certificates(student, course_key, course=None, insecure=False,
         forced_grade - a string indicating to replace grade parameter. if present grading
                        will be skipped.
     """
-    xqueue = XQueueCertInterface()
-    if insecure:
-        xqueue.use_https = False
 
     if not course:
         course = modulestore().get_course(course_key, depth=0)
 
+    users_queryset = list_with_level(course, u'beta')
+    username_list = [user.username for user in users_queryset]
```

Comment:
Why we need a username list? Can we directly check student user object in beta tester users_queryset?